testsuite/features/secondary/trad_centos_migrate_to_min.feature

```diff
@@ -7,8 +7,16 @@ Feature: Migrate a CentOS 7 traditional client into a Salt minion
   As an authorized user
   I want to migrate this CentOS 7 traditional client to a Salt minion
 
-  Scenario: Prepare the CentOS 7 traditional client in the migration context
+  Scenario: Delete the CentOS minion in the migration context
```

Comment:
As a matter of fact, we should be more vague everywhere:
* SLES and openSUSE clients should be "SUSE" or SUSE-like
* Debian and Ubuntu clients should be "Debian" or Debian-like
* CentOS, RedHat, and all the other ones should be "RedHat" or Redhat-like

But that's a big change to the testsuite and let's not start it now. Ping @srbarrios .

Reply:
Indeed, let's not do this refactor in this PR, I feel it's out of the scope.
Let's keep as it is for now.

Reply:
And it's HUGE

I already did it on sumaform BTW, we now have "generic" OS types

Reply:
Thanks for the remarks.